wsmaster/che-core-api-machine/src/main/java/org/eclipse/che/api/machine/server/event/MachineStateMessenger.java

```diff
@@ -44,7 +44,7 @@ public MachineStateMessenger(EventService eventService) {
     public void onEvent(MachineStatusEvent event) {
         try {
             final ChannelBroadcastMessage bm = new ChannelBroadcastMessage();
-            bm.setChannel("machine:status:" + event.getWorkspaceId() + ':' + event.getMachineName());
+            bm.setChannel("workspace:" + event.getWorkspaceId() + ":machines_statuses");
```

Comment:
This kind of refactoring always effects client, and you always change it, and usually it is changed in 2 places(ide, dashboard). I think in this case it will be useful to extract the creation of the channel name into separate SHARABLE class which may be reused by GWT, in that way you don't need to refactor ide client when you change the format of the channel. wdyt?